binary-array-ld-cli/src/test/kotlin/net/bald/BinaryArrayConvertCliTest.kt

```diff
@@ -56,12 +63,59 @@ class BinaryArrayConvertCliTest {
 
     @Test
     fun run_withUri_withOutputFile_outputsToFile() {
-        val inputFile = convertToNetCdf("/netcdf/identity.cdl")
+        val inputFile = writeToNetCdf("/netcdf/identity.cdl")
         val outputFile = createTempFile()
         run("--uri", "http://test.binary-array-ld.net/example", inputFile.absolutePath, outputFile.absolutePath)
 
         val model = createDefaultModel().read(outputFile.toURI().toString(), "ttl")
         ModelVerifier(model).apply {
+            resource("http://test.binary-array-ld.net/example/") {
+                statement(RDF.type, BALD.Container)
+                statement(BALD.contains, model.createResource("http://test.binary-array-ld.net/example/var0")) {
+                    statement(RDF.type, BALD.Resource)
+                }
+                statement(BALD.contains, model.createResource("http://test.binary-array-ld.net/example/var1")) {
+                    statement(RDF.type, BALD.Resource)
+                }
+            }
+        }
+    }
+
+    @Test
+    fun run_withPrefixMappingGroup_outputsPrefixMapping() {
+        val inputFile = writeToNetCdf("/netcdf/prefix.cdl")
+        val outputFile = createTempFile()
+        run("--uri", "http://test.binary-array-ld.net/example", inputFile.absolutePath, outputFile.absolutePath)
+
+        val model = createDefaultModel().read(outputFile.toURI().toString(), "ttl")
```

Comment:
Might be nice here to share the model verification of run_withPrefixMappingGroup_outputsPrefixMapping() and run_withPrefixMappingVar_outputsPrefixMapping() in a separate function (if possible) to make it clearer that both group and var prefix mapping are expecting the same output?

Reply:
Good call, done.